src/NewsService.Mappers/Models/NewsInfoMapper.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+using LT.DigitalOffice.Models.Broker.Responses.Company;
+using LT.DigitalOffice.Models.Broker.Responses.User;
+using LT.DigitalOffice.NewsService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.NewsService.Models.Db;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.NewsService.Mappers.Models
+{
+  public class NewsInfoMapper : INewsInfoMapper
+  {
+    private IRequestClient<IGetUserDataRequest> _userRequestClient;
+    private IRequestClient<IGetDepartmentRequest> _departmentRequestClient;
+    private readonly ILogger _logger;
+
+    private string GetUserFullName(Guid userId)
+    {
+      string fullName = null;
+
+      try
+      {
+        var request = IGetUserDataRequest.CreateObj(userId);
+        var response = _userRequestClient.GetResponse<IOperationResult<IGetUserDataResponse>>(request).Result;
+
+        if (!response.Message.IsSuccess)
+        {
+          _logger.LogWarning($"Can't found user FIO. Reason: '{string.Join(',', response.Message.Errors)}'");
+        }
+        else
+        {
+          fullName = $"{response.Message.Body.LastName} {response.Message.Body.FirstName} {response.Message.Body.MiddleName}".Trim();
+        }
+      }
+      catch (Exception exception)
+      {
+        _logger.LogError(exception, "Exception on get user FIO data request.");
+      }
+
+      return fullName;
+    }
+
+    private string GetDepartmentName(Guid departmentId)
+    {
+      string name = null;
+
+      try
+      {
+        var request = IGetDepartmentRequest.CreateObj(null, departmentId);
```

Comment:
wait for company update or we have to update this service after update company, because consumers and models were updated